README.md

```diff
@@ -61,76 +69,104 @@ _refinery_ is a multi-repository project, you can find all integrated services i
 
 ### Integrations
+
 Lastly, _refinery_ supports [SDK actions](https://github.com/code-kern-ai/refinery-python) like pulling and pushing data. Data-centric AI redefines labeling to be more than a one-time job by giving it an iterative workflow, so we aim to give you more power every day by providing end-to-end capabilities, growing the large-scale availability of high-quality training data. Use our SDK to program integrations with your existing landscapes.
 
+## Your benefits
 
-### Your benefits
-You gain better insights into the data labeling workflow, receive an implicit documentation for your training data (which you can use to discuss findings), and can ultimately build better models in shorter time.
+You can automate tons of repetitive tasks, gain better insights into the data labeling workflow, receive an implicit documentation for your training data, and can ultimately build better models in shorter time.
 
 Our goal is to make training data building feel more like a programmatic and enjoyable task, instead of something tedious and repetitive. _refinery_ is our contribution to this goal. And we're constantly aiming to improve this contribution.
 
-If you like what we're working on, please leave a ⭐! 
+If you like what we're working on, please leave a ⭐!
+
+## How does Kern AI make money, if refinery is open-source?
+
+You won't believe how often we get that question - and it is a fair one :) Put short, the open-source version of _refinery_ is currently a single-user version, and you can get access to a multi-user version in our commercial version. Additionally, we have commercial products on top of _refinery_, e.g. to use the _refinery_ automations as an actual realtime prediction API.
+
+Generally, we are passionate about open-source and want to contribute as much as possible.
 
 ## 🤓 Features
+
 For a detailed overview of features, please look into our [docs](https://docs.kern.ai).
 
 ### (Semi-)automated labeling workflow for NLP tasks
+
 - Both manual and programmatic for classifications and span-labeling
 - Integration with state-of-the-art libraries and frameworks
 - Creation and management of lookup lists/knowledge bases to support during labeling
 - Neural search-based retrieval of similar records and outliers
 - Sliceable labeling sessions to drill-down on specific subsets
 - Multiple labeling tasks possible per project
+- Rich library of ready-made automations in our open-source [bricks](https://github.com/code-kern-ai/bricks) library
 
 ### Extensive data management and monitoring
+
```

Comment:
i would have expected a note to our data browser since this is data management?

Reply:
makes sense, added that